config/config.go

```diff
@@ -80,6 +80,7 @@ const (
 	IstioAppLabel             = "app"                                    // we can assume istio components are labeled with "app"
 	IstioRevisionLabel        = "istio.io/rev"                           // the standard label key used to identify the istio revision.
 	IstioVersionLabel         = "version"                                // we can assume istio components are labeled with "version", if versioned
+	KubernetesIstioLabel      = "app.kubernetes.io/name"
```

Comment:
nit: this isn't istio specific
```suggestion
	KubernetesAppLabel      = "app.kubernetes.io/name"
```